tests/foreman/endtoend/test_api_endtoend.py

```diff
@@ -153,7 +153,7 @@
         '/katello/api/capsules/:id/content/sync',
         '/katello/api/capsules/:id/content/sync',
         '/katello/api/capsules/:id/content/sync',
-        '/katello/api/capsules/:id/reclaim_space',
+        '/katello/api/capsules/:id/content/reclaim_space',
```

Comment:
@vsedmik this update was already included in @pondrejk 's [PR](https://github.com/SatelliteQE/robottelo/pull/13857) which updates other endpoints too, so are we good closing this PR now?

Reply:
Oh, how dare you!! :D